File: 1446-consecutive-characters/1446-consecutive-characters.c
int maxPower(char * s){
    int max = 1;
    int tmp = 1;
    int i = 0;
    
    while(s[i])
    {
        if(s[i+1] && s[i] == s[i+1])
            tmp++;
        else{
            if(tmp > max)
                max = tmp;
            tmp = 1;
        }
        i++;
    }
    return max;
}

File: 1446-consecutive-characters/1446-consecutive-characters.java
class Solution {
    public int maxPower(String s) {
        int max = 1, tmp = 1;
        
        for(int i = 1; i < s.length(); i++){
            if(s.charAt(i-1) == s.charAt(i)){
                tmp++;
                if(tmp > max)
                    max = tmp;
            }
            else
                tmp = 1;
        }
        return max;
    }
}